plaso/parsers/text_parser.py

```diff
@@ -251,6 +251,24 @@ def __init__(self):
     # a structural fix.
     self._line_structures = list(self.LINE_STRUCTURES)
 
+  def _GetValueFromStructure(self, structure, name, default_value=None):
```

Comment:
Is this additional level of indirection really needed?
I feel like having tructure.get('hostname', None) would be explicit enough (as this is also how we access dict values)

Reply:
the reason for this is the docstring is used to explain why this is needed, instead of putting comments in the different parsers
IMHO this is one of those things that is easy to forget

Reply:
SGTM